atlasdb-client/src/main/java/com/palantir/atlasdb/stream/AbstractGenericStreamStore.java

```diff
@@ -156,7 +156,7 @@ private File loadToNewTempFile(Transaction transaction, ID id, StreamMetadata me
         }
     }
 
-    private void checkStreamStored(ID id, StreamMetadata metadata) {
+    protected void checkStreamStored(ID id, StreamMetadata metadata) {
```

Comment:
Removed this one, since it's no longer needed, but added comments in other similar cases.